src/components/PeopleItem/PeopleItem.tsx

```diff
@@ -0,0 +1,63 @@
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  person: Person;
+  personId: string;
+  people: Person[];
+};
+export const PeopleItem: React.FC<Props> = ({ person, personId, people }) => {
+  const isSelected = (human: Person) => human.slug === personId;
+
+  const motherFind = people.find(
+    ({ name: personName }) => personName === person.motherName,
+  );
+
+  const fatherFind = people.find(
+    ({ name: personName }) => personName === person.fatherName,
+  );
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': isSelected(person),
+      })}
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={classNames({
+            'has-text-danger': person.sex === 'f',
+          })}
+        >
+          {person.name}
+        </Link>
```

Comment:
from readme: create the `<PersonLink person={person} />` and use it for all existing people;